### ISSUE TITLE
Sleeping thread shown as running in JFR snapshot

### ISSUE THREAD

**Issue by BaesKevin** [oracle/visualvm#250]
Steps to reproduce:

* create a SleepyThread class
```java
    public static void main(String[] args) throws InterruptedException {
        Thread.sleep(10000);
    }
```
* run with `java -XX:+UnlockDiagnosticVMOptions -XX:+DebugNonSafepoints -XX:+FlightRecorder -XX:StartFlightRecording=name="sleepy thread",settings="profile",dump
onexit=true,filename="sleepythread.jfr" SleepyThread.java`
* open the recording in VisualVM, the threads tab will show the main thread 'running' for 10 seconds. 
The same recording works as expected in JMC 7.0.2: the main thread is shown as sleeping 10 seconds.

![visualvm output](https://user-images.githubusercontent.com/26406179/99190452-e231e400-2766-11eb-919b-bf4583bdc0f3.png)
![JMC output](https://user-images.githubusercontent.com/26406179/99190495-1a392700-2767-11eb-9770-ca391a1a7030.png)

System:
* Windows version 1803
* VisualVM: 2.0.4
* java -version output
```
openjdk version "15.0.1" 2020-10-20
OpenJDK Runtime Environment (build 15.0.1+9-18)
OpenJDK 64-Bit Server VM (build 15.0.1+9-18, mixed mode, sharing)
```

[sleepythread_logfile.txt](https://github.com/oracle/visualvm/files/5545010/sleepythread_logfile.txt)

**Comment by thurka** [comment]
Thanks for report. Can you please attach VisualVM log file, so we can diagnose the issue? It loos like duplicate of GH-222, but without VisualVM log file it is hard to tell.

**Comment by BaesKevin** [comment]
Updated issue with logfile. Exported logfile right after opening the snapshot and opening the threads tab.

**Comment by jisedlac** [comment]
Duplicate of GH-222, fixed in VisualVM 2.0.5.
